spec/integration/support/kiwi_export_examples.rb

```diff
@@ -15,27 +15,38 @@
 # To contact SUSE about this file by physical or electronic mail,
 # you may find current contact information at www.suse.com
 
-shared_examples "kiwi export" do
+shared_examples "kiwi export" do |opts|
+  let(:options) { opts || {
+      machinery_dir: "/home/vagrant/.machinery",
+      owner: "vagrant",
+      group: "vagrant"
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.